src/ws/session.c

```diff
@@ -544,10 +544,9 @@ perform_tlscert (const char *rhost,
 
   debug ("start tls-cert authentication for cockpit-ws %u", getppid ());
 
-  if (strncmp (authorization, "tls-cert ", 9) == 0)
-    client_certificate = authorization + 9;
-  else
-    exit_init_problem (PAM_AUTH_ERR);
+  /* True, otherwise we wouldn't be here. */
+  assert (strncmp (authorization, "tls-cert ", 9));
```

Comment:
Before you wonder about all sorts of failures: Missing `== 0`

Reply:
Thanks lol.